### PR TITLE
Feature/add documentation

### PR DESCRIPTION
**Pull request by joonasnuutinen**
This should be clear enough for a new developer to join the project and also for a new organization to deploy it.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,3 +8,6 @@ UPRIGHT_API_TOKEN=
 SLACK_TOKEN=
 # From Slack channel preferences -> About -> Channel ID (Scroll to the bottom if not immediately visible)
 SLACK_CHANNEL=
+SLACK_ERROR_CHANNEL=
+
```
Comment (MelissaAlasalmi) [inline .env.example:12]: Should we also add the login creds needed for the internal API?  :)

**Review by MelissaAlasalmi — APPROVED** [review]
One of the best READMEs I have seen! 💪 
Only thing I would add, maybe, is a bit of text about the "why" - what kind of info you get from the profile, etc, and how it can be a good conversation starter.

File: .env.example (hunk 1 above)
Comment (joonasnuutinen) [reply .env.example:12]: Yes, feel free to update the file as new environment variables are introduced

**Comment by joonasnuutinen** [comment]
> One of the best READMEs I have seen! 💪 Only thing I would add, maybe, is a bit of text about the "why" - what kind of info you get from the profile, etc, and how it can be a good conversation starter.

Thanks! Yes, that would be a good addition. Let's refine this later.

**Pull request merged**